.gitlab-ci.yml

```diff
@@ -39,3 +39,24 @@ staging:
         K8S_SECRET_MAILER_EMAIL_BACKEND: "anymail.backends.mailgun.EmailBackend"
         K8S_SECRET_DEFAULT_FROM_EMAIL: "no-reply@hel.ninja"
         K8S_SECRET_FIELD_ENCRYPTION_KEYS: "f164ec6bd6fbc4aef5647abc15199da0f9badcc1d2127bde2087ae0d794a9a0b"
+
+production:
+    variables:
+        K8S_SECRET_ALLOWED_HOSTS: "*"
+        K8S_SECRET_DEBUG: 0
+        K8S_SECRET_OIDC_SECRET: "$GL_HEL_STABLE_TUNNISTAMO_OIDC_SECRET"
+        K8S_SECRET_SECRET_KEY: "$GL_STABLE_DJANGO_SECRET_KEY"
+        K8S_SECRET_SENTRY_DSN: "https://c2a1cc12588942fd938d8cfdea617cb4:6fc6a7eb191a48bf9be329ca344d6dbc@sentry.hel.ninja/33"
```

Comment:
While nowadays I think this isn't really considered a strict secret, you can still do at least some harassment with it, so I would not expose it publicly as it is trivial to use a variable instead.